src/types/requests-types.ts

```diff
@@ -0,0 +1,36 @@
+export type PersonRequest = {
+  name: string;
+  height: string;
+  mass: string;
+  hair_color: string;
+  skin_color: string;
+  eye_color: string;
+  birth_year: string;
+  gender: string;
+  homeworld: string;
+  films: string[];
+  species: string[];
+  vehicles: string[];
+  starships: string[];
+  created: string;
+  edited: string;
+  url: string;
+};
+
```

Comment:
Maybe it's better to create more smaller interfaces, combined by common characteristics

Reply:
done ✅